global/prometheus-alertmanager/templates/config.yaml

```diff
@@ -251,7 +251,7 @@ data:
 
       # ---- webhook handler: metal ----
       - receiver: wham_metal
-        continue: false
+        continue: true
```

Comment:
being the last receiver for metal alerts this is `continue: false`

Reply:
ah ok, should it be last one? (or is that mandatory to have a last handler?)...i saw my handler more like a "middleware"..

Reply:
in this case wham is the last receiver. the [dev-null receiver](https://github.com/sapcc/helm-charts/blob/776e03ead9d67d96a6ca5b957cb4d1f3df3e5dbb/global/prometheus-alertmanager/templates/config.yaml#L260-L262) is meant to catch alerts that aren't labeled properly. for this work properly labeled alerts need to have a final receiver before

Reply:
ok got it!